crates/oxc_ast/src/ast/ts.rs

```diff
@@ -1288,6 +1288,13 @@ impl<'a> Modifiers<'a> {
             .map_or(false, |modifiers| modifiers.iter().any(|modifier| modifier.kind == target))
     }
 
+    pub fn find<F>(&self, f: F) -> Option<&Modifier>
```

Comment:
Can't you use the `contains` above?

Reply:
```ts
declare const enum Enum {
  A = 1,
  B,
  C = 2,
}
```

Because I need to know the span of `const` to report and delete in fixer.